File: itv/JCZhang-ITVConsultas-Funcional-JAVA/src/main/java/models/Moto.java
package models;

public class Moto extends Vehicle {
    int cilindrada;

    public Moto(String marca, int fabricationYear, boolean apto, double numKm, int cilindrada){
        super(marca, fabricationYear, apto, numKm);
        this.marca = marca;
        this.fabricationYear = fabricationYear;
        this.apto = apto;
        this.numKm = numKm;
        this.cilindrada = cilindrada;
    }

    @Override
    public String toString() {
        return "Moto{" +
                "marca='" + marca + '\'' +
                ", fabricationYear=" + fabricationYear +
                ", apto=" + apto +
                ", numKm=" + numKm +
                ", cilindrada=" + cilindrada +
                '}';
    }

    public int getCilindrada() {
        return cilindrada;
    }
}

File: itv/JCZhang-ITVConsultas-Funcional-Kotlin/src/main/kotlin/models/Moto.kt
package models

class Moto(marca: String, fabricationYear: Int, apto: Boolean, numKm: Int, val cilindrada: Int): Vehicles(marca, fabricationYear, apto,
    numKm
) {
    override fun toString(): String {
        return "Moto(marca='$marca', fabricationYear='$fabricationYear', apto=$apto, numKm=$numKm, cilindrada = $cilindrada)"
    }

}